src/scheduler/annualReviewStartEmailScheduler.ts

```diff
@@ -0,0 +1,211 @@
+      itemId: list.id,
+    },
+    postEvents[context.daysBeforeAnnualReviewStart].auditEvent as AuditEvent,
+    "list"
+  );
+}
+
+async function emailPosts(
+  listItemsEligibleForAnnualReview: ListItem[],
+  list: List,
+  context: AnnualReviewStartDateContext,
+) {
+
+  if (listItemsEligibleForAnnualReview?.length > 0 && list.jsonData.publishers) {
+    for (const publisherEmail of list.jsonData.publishers) {
+      const annualReviewPostEmailResult = await sendAnnualReviewPostEmail(
+        context.daysBeforeAnnualReviewStart,
+        publisherEmail,
+        lowerCase(startCase(list.type)),
+        list?.country?.name ?? "",
+        list.nextAnnualReviewStartDate.toDateString()
+      );
+
+      if (annualReviewPostEmailResult.error) {
+        logger.error(`Unable to send annual review email to post contact ${publisherEmail}`);
+      }
+    }
+    await addAuditEventForEmail(context, list);
+  }
+}
+
+async function emailProviders(updatedListItems: ListItem[], list: List, context: AnnualReviewStartDateContext) {
+  for (const updatedListItem of updatedListItems) {
+    // @TODO update correct landing page once latest annual review changes are merged in
+    const formRunnerEditUserUrl = await initialiseFormRunnerSession(
+      list,
+      updatedListItem as ListItemGetObject,
+      "update your annual review",
+      false
+    );
+
+    const providerEmailResult = await sendAnnualReviewProviderEmail(
+      context.daysBeforeAnnualReviewStart,
+      (updatedListItem.jsonData as BaseDeserialisedWebhookData).emailAddress,
+      lowerCase(startCase(updatedListItem.type)),
+      list?.country?.name ?? "",
+      (updatedListItem.jsonData as BaseDeserialisedWebhookData).contactName,
+      context.annualReviewStartDate.toDateString(),
+      formRunnerEditUserUrl
+    );
+
+    if (providerEmailResult.result) {
+      await recordListItemEvent(
+        {
+          eventName: "sendAnnualReviewStartedProviderEmail",
+          itemId: list.id,
+        },
+        AuditEvent.ANNUAL_REVIEW_STARTED_PROVIDER_EMAIL_SENT,
+        "listItem"
+      );
+    }
+  }
+}
+
+async function sendEmails(lists: List[], context: AnnualReviewStartDateContext): Promise<void> {
+  logger.info(`[${lists.length}] lists identified ${context.daysBeforeAnnualReviewStart}, days before annual review starts ${JSON.stringify(lists)}`);
+
+  const listIds = lists.map(list => list.id);
+  const findListItemsResult = await findListItemsForLists(listIds, ["PUBLISHED", "CHECK_ANNUAL_REVIEW"], false);
+  if (findListItemsResult.error) {
+    logger.error(`Unable to send emails.  Unable to retrieve List Items for Lists ${listIds}`);
+
+  } else if (findListItemsResult.result) {
+    const listItemsForAllLists = findListItemsResult.result;
+    for (const list of lists) {
+      // get list items eligible for annual review
+      const listItemsEligibleForAnnualReview = listItemsForAllLists.filter(listItem => listItem.listId === list.id);
```

Comment:
what if you change 
`findListItemsForLists` so it does a `prisma.list.findMany()`?

then when you iterate through lists, you don't have to iterate again to get the relevant list items. They are contained within the list